identity-service/src/routes/healthCheck.js

```diff
@@ -191,13 +191,16 @@ module.exports = function (app) {
       req.logger.error(`Failed to fund relayer with error: ${err}`)
     }
 
-    for (let account of RELAY_HEALTH_ACCOUNTS) {
-      let balance = parseFloat(Web3.utils.fromWei(await getRelayerFunds(account), 'ether'))
-      balances.push({ account, balance })
-      if (balance < minimumBalance) {
-        belowMinimumBalances.push({ account, balance })
-      }
-    }
+    balances = await Promise.all(
+      [RELAY_HEALTH_ACCOUNTS].map(async account => {
```

Comment:
I think we need to spread these accts like 
```
[...RELAY_HEALTH_ACCOUNTS]
```

Reply:
actually, we might not even need to spread it and can use that variable directly since it's already in a list

Reply:
ah it's because RELAY_HEALTH ACCOUNTS is in a set. in that case, let's do what I suggested in the first comment and ignore the second :)

Reply:
Oops didn't realize what the spread operator was. Added it.